integration/apps/nekbone/nekbone.py

```diff
@@ -76,7 +76,10 @@ def trial_setup(self, run_id, output_dir):
 
     def experiment_teardown(self, output_dir):
         file_name = os.path.join(output_dir, self._input_name)
-        os.unlink(file_name)
+        try:
+            os.unlink(file_name)
+        except FileNotFoundError:
```

Comment:
FileNotFoundError does not exist in Python 2.  You can use IOError instead for backwards compatability.

Sorry I didn't catch this on my first run through.  I was bitten by this exact issue yesterday, so now it's fresh in my mind. :)